### PR TITLE
Adding Consumer Group to the EventHub binding

### PR DESCRIPTION
**Pull request by mathewc**
Support for EH consumer group is going out in the upcoming release, so we need to expose this.

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -89,6 +89,14 @@
...
                     "help": "This is the connection string for your event hub. Make sure that it has Receive permissions.",
                     "placeholder": "[variables('selectConnection')]"
+                },
+                {
+                    "name": "consumerGroup",
+                    "value": "string",
+                    "defaultValue": "$Default",
+                    "required": false,
+                    "label": "Event Hub consumer group",
+                    "help": "The name of the Event Hub consumer group to receive events from."
```
Comment (mathewc) [inline Bindings/bindings.json]: Need to find out what if any validator can be applied. Until we know for sure, we won't have one.

File: Templates/EventHubTrigger-CSharp/function.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
             "name": "myEventHubMessage",
             "direction": "in",
             "path": "samples-workitems",
-            "connection": ""
+            "connection": "",
+            "consumerGroup": "$Default"
```
Comment (mathewc) [inline Templates/EventHubTrigger-CSharp/function.json]: The runtime will default this if not specified. Should I put these default values here or remove them and let the template take care of it?

File: Bindings/bindings.json (hunk 1 above)
Comment (mathewc) [inline Bindings/bindings.json]: Got confirmation from the EH team that we can just use the Storage naming rules, and whitelist $Default

File: Templates/EventHubTrigger-CSharp/function.json (hunk 2 above)
Comment (soninaren) [inline Templates/EventHubTrigger-CSharp/function.json]: Typically non mandatory fields are not added to templates, if the customer needs that explicitly they can add the value from integration tab, which then will just add the setting to function.json

**Comment by soninaren** [comment]
:shipit:

File: Templates/EventHubTrigger-CSharp/function.json (hunk 2 above)
Comment (mathewc) [inline Templates/EventHubTrigger-CSharp/function.json]: Ok, I'll remove these defaults.

**Pull request merged**